security/tools/sds/sds_cli.go

```diff
@@ -69,6 +79,7 @@ func sdsRequest(socket string, req *xdsapi.DiscoveryRequest) *xdsapi.DiscoveryRe
 
 func main() {
 	sdsSocket := flag.String("socket", "/tmp/sdsuds.sock", "SDS socket")
+	caFile := flag.String("caFile", "./../../docker/nodeagent-sds-cert.pem", "SDS socket")
```

Comment:
This is just a self signed certificate, not CA(able to issue certificate). Rename it to "sdsCert"?

Reply:
The flag description is also incorrect.

Reply:
updated.